exporter/cat1_view_helper.go

```diff
@@ -87,16 +87,16 @@ func entryInfosForPatient(patient models.Record, measures []models.Measure, vsMa
 		for i, entrySection := range entrySections {
 			if entrySection != nil {
 				entry := entrySections[i].GetEntry()
-				SetCodeDisplaysForEntry(entry)
+				SetCodeDisplaysForEntry(entry, mappedDataCriteria)
 			}
 		}
 		entryInfos = appendEntryInfos(entryInfos, entrySections, mappedDataCriteria)
 	}
 	return entryInfos
 }
 
-func SetCodeDisplaysForEntry(e *models.Entry) {
-	codeDisplays := codeDisplayForQrdaOid(HqmfToQrdaOid(e.Oid))
+func SetCodeDisplaysForEntry(e *models.Entry, mapDataCriteria mdc) {
```

Comment:
Do we think this method should accept the `mdc` type, then pull the `ValueSetOid` off it, or should it just accept a `string` and we do the `mapDataCriteria.dcKey.ValueSetOid` in the caller?